packages/firestore/src/api/database.ts

```diff
@@ -429,9 +429,11 @@ follow these steps, YOUR APP MAY BREAK.`);
   }
 
   INTERNAL = {
-    delete: async (): Promise<void> => {
+    delete: async (options?: {
```

Comment:
Note that I did not expose this option via the .d.ts definition for delete().

Reply:
I played around with this more yesterday - and figured out why this isn't working (which sounds like both good and bad news, but is probably just bad news). If I pass ":memory:" or "file::memory:" to the IndexedDB Shim, it tries to create a file on disk with that name.. and subsequently times out all operations.  Since setting `basepath` also doesn't work for me ("The upgrade transaction was aborted."), I am inclined to keep this cleanup. Without it, a simple run creates hundreds of files in the main repo directory - which looks fairly scary for users that don't know what's going on.

I also think that we should do this cleanup for our tests in general - without it, we are leaving a ton of data behind in Chrome headless.

I can delete the lingering files in `withTestDb()` and not rely on `.shutdown()` if you do prefer this. We would then have to special case this testing mode there as well.